hubtraf/auth/lti.py

```diff
@@ -1,20 +1,32 @@
-from oauthlib.oauth1.rfc5849 import signature
 import time
 import uuid
+
+from oauthlib.oauth1.rfc5849 import signature
+
 from hubtraf.user import OperationError
 
 
-async def lti_login_data(session, log, hub_url, username, consumer_key, consumer_secret, launch_url, extra_args={}):
+async def lti_login_data(
+    session,
+    log,
+    hub_url,
+    username,
+    consumer_key,
+    consumer_secret,
+    launch_url,
+    extra_args={},
```

Comment:
We should eventually default this to `None` (because mutable defaults in python args are oof) but doesn't need to be a part of this PR

Reply:
Resolved!